app/models/runtime/asg_latest_update.rb

```diff
@@ -8,9 +8,9 @@ class AsgTimestamp < Sequel::Model
     def self.renew
       old_update = AsgTimestamp.first
       if old_update
-        old_update.update(last_update: Sequel::CURRENT_TIMESTAMP)
+        old_update.update(last_update: Time.now)
```

Comment:
```suggestion
        old_update.update(last_update: Time.now.utc)
```

Can we explicitly make this be in UTC? It probably is in practice

Reply:
Fixed